presto-main/src/main/java/com/facebook/presto/metadata/FunctionAndTypeManager.java

```diff
@@ -363,9 +386,19 @@ public List<SqlFunction> listFunctions(Session session, Optional<String> likePat
         ImmutableList.Builder<SqlFunction> functions = new ImmutableList.Builder<>();
         if (!isListBuiltInFunctionsOnly(session)) {
             functions.addAll(SessionFunctionUtils.listFunctions(session.getSessionFunctions()));
-            functions.addAll(functionNamespaceManagers.values().stream()
-                    .flatMap(manager -> manager.listFunctions(likePattern, escape).stream())
-                    .collect(toImmutableList()));
+
+            // If coordinator sidecar is enabled, filter on the current default namespace
+            if (coordinatorSidecarEnabled) {
```

Comment:
In case the coordinator sidecar isn't enabled, wouldn't this be the same thing?  Can we remove this branch?